### PR TITLE
Sdk 2.0 i18 changes

### PR DESCRIPTION
**Pull request by BharadwajAd-Kore**
FLY-7441

### CODE REVIEW

File: src/components/search/findly-sdk.js [hunk 1]
```diff
@@ -1800,7 +1806,7 @@ FindlySDK.prototype.getSearchTemplate = function (type) {
...
                                   <span class="pr_15">${pageInfo.feedbackAppearances} Views</span>\
                                   <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA0AAAAOCAYAAAD0f5bSAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAADnSURBVHgBnZHRDYJADIZ7hwk8OoJuoBvgBjqB4ZEnRgA28I3AC2ECdQNGcAMZwQXg/KuHQTxOY5Om0OvXa/8TNLI8z+Ou6xJ8JmEYpmQwOU5oYMkQTZg0JXFDQxaT9IfNbIdZlimEGu7TYEcrRM/drn1Ek4RhyWpxR3g8JvrdBjs+BBIMQLG14zilUuqkxxBkGZmFSDWwE0Js6QeT6JoAOAM4avDyFdIzv4FTxUVRrBCa1zsNwbIs5yaobdsI+1cfC2tZ93B+k5qVQ26BZhH+fdd1N0aVUORzEbpy5FsbeOV53iEIgtsdmbJxSEqhuZgAAAAASUVORK5CYII=">\
-                                  <span>${pageInfo.feedbackClicks} Clicks</span>\
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js]: @BharadwajAd-Kore : Could you please check this you removed pageInfo feedback clicks data
@SunilSi-kore

File: src/templatemanager/templates/finalResultsTemplate/finalResultsTemplate.ts [hunk 2]
```diff
@@ -136,7 +141,7 @@ class FinalResultsTemplate {
             <div class="top-header">\
                 <div class="top-header-with-img">\
                     <span class="logo-span"><img src="https://koregeneric.s3.amazonaws.com/SearchAssist_UI_Img/snippet_imgs/snippet-avathar.svg"/></span>\
-                    <div class="btn-chip">SUGGESTED ANSWER</div>\
+                    <div class="btn-chip sdk-i18n-lang" sdk-i18n-key="sa_sdk_Suggested_answer">{{html langTranslator("sa_sdk_Suggested_answer")}}</div>\
                 </div>\
                 {{if snippetData && snippetData.snippet_type === "generative_model"}}\
                 <div class="btn-link"><span class="bot-bg-purple"><img src="https://koregeneric.s3.amazonaws.com/SearchAssist_UI_Img/snippet_imgs/bot.svg"/></span>ANSWERED BY AI</div>\
```
Comment (VenkatIc-Kore) [inline src/templatemanager/templates/finalResultsTemplate/finalResultsTemplate.ts]: @BharadwajAd-Kore : i18n changes missed
@SunilSi-kore

File: src/templatemanager/templates/finalResultsTemplate/finalResultsTemplate.ts [hunk 3]
```diff
@@ -183,9 +188,9 @@ class FinalResultsTemplate {
         <div class="top-header">\
             <div class="top-header-with-img">\
                 <span class="logo-span"><img src="https://koregeneric.s3.amazonaws.com/SearchAssist_UI_Img/snippet_imgs/snippet-avathar.svg"/></span>\
-                <div class="btn-chip">SUGGESTED ANSWER</div>\
-            </div>\
-            {{if snippetData && snippetData.snippet_type === "generative_model"}}\
+                <div class="btn-chip sdk-i18n-lang" sdk-i18n-key="sa_sdk_Suggested_answer">{{html langTranslator("sa_sdk_Suggested_answer")}}</div>\
+                </div>\
+            {{if snippetData && snippetData.source === "Answered by AI"}}\
             <div class="btn-link"><span class="bot-bg-purple"><img src="https://koregeneric.s3.amazonaws.com/SearchAssist_UI_Img/snippet_imgs/bot.svg"/></span>ANSWERED BY AI</div>\
```
Comment (VenkatIc-Kore) [inline src/templatemanager/templates/finalResultsTemplate/finalResultsTemplate.ts]: @BharadwajAd-Kore : ANSWERED BY AI i18n changes missed
@SunilSi-kore

File: src/templatemanager/templates/fullsearchResultsTemplate/fullsearchResultsTemplate.ts [hunk 4]
```diff
@@ -464,8 +468,7 @@ class FullSearchResultsTemplate {
                         {{/if}}\
                     {{/each}}\
                     {{if searchFacet.multiselect}}\
-                    <div class="apply-btn">Apply</div>\
-                    {{/if}}\
+                    <div class="apply-btn sdk-i18n-lang" sdk-i18n-key="sa_sdk_apply">{{html langTranslator("sa_sdk_apply")}}</div>\                    {{/if}}\
                       {{if !searchFacet.multiselect}}\
                       <div class="clear-all sdk-clear-all-facet-top">Clear</div>\
```
Comment (VenkatIc-Kore) [inline src/templatemanager/templates/fullsearchResultsTemplate/fullsearchResultsTemplate.ts]: @BharadwajAd-Kore : i18n change missed
@SunilSi-kore

File: src/templatemanager/templates/searchListViewTemplate/searchListViewTemplate.ts [hunk 5]
```diff
@@ -621,23 +622,23 @@ class SearchListViewTemplate {
...
-                  <button class="next-btn sdk-tours-info-nxt">Next</button>\
-                  <button class="close-btn sdk-tours-info-close">Close</button>\
-              </div>\
+              <button class="next-btn sdk-tours-info-nxt sdk-i18n-lang" sdk-i18n-key="sa_sdk_next">{{html langTranslator("sa_sdk_next")}}</button>\
+              <button class="close-btn sdk-tours-info-close sdk-i18n-lang" sdk-i18n-key="sa_sdk_close">{{html langTranslator("sa_sdk_close")}}</button>\
+             </div>\
             </div>\
           </div>\
           <div class="tours-information tour-customization-info sdk-tours-info-end hide">\
             <div class="tourtitle">Customize</div>\
```
Comment (VenkatIc-Kore) [inline src/templatemanager/templates/searchListViewTemplate/searchListViewTemplate.ts]: @BharadwajAd-Kore : customize i18n change missed
@SunilSi-kore

File: src/templatemanager/templates/snippetListTemplate/snippetListTemplate.ts [hunk 6]
```diff
@@ -27,9 +27,9 @@ class SnippetListTemplate {
         <div class="top-header">\
             <div class="top-header-with-img">\
                 <span class="logo-span"><img src="https://koregeneric.s3.amazonaws.com/SearchAssist_UI_Img/snippet_imgs/snippet-avathar.svg"/></span>\
-                <div class="btn-chip">SUGGESTED ANSWER</div>\
-            </div>\
-            {{if snippetData && snippetData.snippet_type === "generative_model"}}\
+                <div class="btn-chip sdk-i18n-lang" sdk-i18n-key="sa_sdk_Suggested_answer">{{html langTranslator("sa_sdk_Suggested_answer")}}</div>\
+                </div>\
+            {{if snippetData && snippetData.source === "Answered by AI"}}\
             <div class="btn-link"><span class="bot-bg-purple"><img src="https://koregeneric.s3.amazonaws.com/SearchAssist_UI_Img/snippet_imgs/bot.svg"/></span>ANSWERED BY AI</div>\
```
Comment (VenkatIc-Kore) [inline src/templatemanager/templates/snippetListTemplate/snippetListTemplate.ts]: @BharadwajAd-Kore : ANSWERED BY AI  key don't have i18n changes , could u please check  and replace 18n change in all the places
@SunilSi-kore

File: src/templatemanager/templates/snippetParagraphTemplate/snippetParagraphTemplate.ts [hunk 7]
```diff
@@ -27,9 +28,9 @@ class SnippetParagraphTemplate {
       <div class="top-header">\
           <div class="top-header-with-img">\
               <span class="logo-span"><img src="https://koregeneric.s3.amazonaws.com/SearchAssist_UI_Img/snippet_imgs/snippet-avathar.svg"/></span>\
-              <div class="btn-chip">SUGGESTED ANSWER</div>\
-          </div>\
-          {{if snippetData && snippetData.snippet_type === "generative_model"}}\
+              <div class="btn-chip sdk-i18n-lang" sdk-i18n-key="sa_sdk_Suggested_answer">{{html langTranslator("sa_sdk_Suggested_answer")}}</div>\
+              </div>\
+          {{if snippetData && snippetData.source === "Answered by AI"}}\
           <div class="btn-link"><span class="bot-bg-purple"><img src="https://koregeneric.s3.amazonaws.com/SearchAssist_UI_Img/snippet_imgs/bot.svg"/></span>ANSWERED BY AI</div>\
```
Comment (VenkatIc-Kore) [inline src/templatemanager/templates/snippetParagraphTemplate/snippetParagraphTemplate.ts]: @BharadwajAd-Kore : i18n change missed
@SunilSi-kore

File: src/components/search/findly-sdk.js [hunk 8]
```diff
@@ -16,8 +16,11 @@ import './css/common-styles.scss';
 import './css/search-bar-experience.scss';
 import './css/fonts/inter.css';
 import '../../libs/perfectscroll/css/perfect-scrollbar.min.css';
+import enJsonObj from '../../libs/languageConversion/i18n/en.json'
+import jaJsonObj from '../../libs/languageConversion/i18n/ja.json'
+import koJsonObj from '../../libs/languageConversion/i18n/ko.json'
+import './css/lang.scss'
 // import "../../../node_modules/jquery-ui/dist/jquery-ui.min";
-import '../../../node_modules/jquery-ui/ui/widgets/draggable.js';
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js:20]: @BharadwajAd-Kore : don't removed draggable.js which is for avatar experience 
@SunilSi-kore

File: src/components/search/findly-sdk.js [hunk 9]
```diff
@@ -1552,7 +1556,7 @@ FindlySDK.prototype.getSearchTemplate = function (type) {
...
...XR4QmRBdjFEZXMvTU11YlVZOUt0ZHFsb0JNdWhCemswamlIaW1hQm5nT1pweU1CRzFvbHZVUVRoaGg3bGdPNDRud0hoSVM5czdMVTFGVW5vcFBDSHpPWWN5U1JYNmtWdGhkcU5TbFlkOThvdWlGaHUwUUk2R1VXNDBrL3JjNG9mOUVWcDFscUZ5T0RoRHRTTGR4Uk5QcHY2R2NvQUwwTTdyKzJvdGxiWEtyc1JCSXUxd1JqZDljRmFJc0RoVGpmVVh5MnZ3NlBIR1ZucDdNRnc3dnQxdjRmc0p5QWszMjZ2L2hvbmd4TmdEMXRjUjhqNFFJM1JLN01PZFFvUERhbWRZK0lmU3Y3UUFuOEV4OEtjenVmRXh2TUQ0SkZjbERjMFBTbDRJN0xrN1ZHczBpSFFDSGVwN0I5SldubE0rc0ZSeDVJdjBUNG5BVlR6TGJMRTRON3lIM045RWdXOHNkU0FjbGMyOElCM2xwLzlKR21JNDhWU2E2cDF6RmVNOFJDVEIvaVh0UDRodm9VUk9Na0ZFeklNQ2pSb0wydmE1TmQzM2kxZldmM044clh0UHlxSEIxdUtseGFWbkRRZExJZjFVWWEvcFhRSzV5ZEhQZVV4clFjLzVRVjUvT3hMellSQ1BZaXZkTjdidjFkMkQzZWpZNjVxZ1NXZE14NGowQ1pwaTdwUEphRkwvYktwMHlUMUN3VUFJcWdnajRITzhhRGdHVkxtbWJRZmxNWkY0M2M2K2FKSnorbWRJSUZzVDlsK0hxWWRuVUQycVEvdSs4ZTBkNldSblgzR0EwOWZZT0p6Z09sREwrNDZ0Y0tVSzhiME5yVFNyOTJyd1d4M2VYMmU3MTdNS3R1RGxPSzg3YW5uRGZySFMvWWd1WjRQMS9YRGViaEd1NFpob0tsSzZGeExXdWQ2SC93RlR1WUQ3OE1qUEJtTzI5b0Q2TTY5ZS9wazk1WU1yOXBHbG1RODRJWFFxL1BQTzh3bGNJNFc0QTZ3Yy9RdGVuWDUzOWUvcUk5c3ZWQStxbTlrWEZ0K3JieTVraDJkbWRHaUxGeU1DWjZ6YmFtUkxXckdpTkh3OHRLaTFrSGtYQnRQdzMwOTZOL1dQWnRWMFdpMzYvTnIyQ0VrM3MyUEdUeldMaHgwMW5GdDhGNWpRSm90R1RkWTRuUVl2bXpCSENjc2N5VTlqQmw4eGkzU
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js:1559]: @BharadwajAd-Kore : could you please check message was changed
@SunilSi-kore

File: src/components/search/findly-sdk.js [hunk 10]
```diff
@@ -5768,14 +5766,29 @@ FindlySDK.prototype.searchEventBinding = function (
...
           $(".skelton-load-img").show();
           $("#live-search-result-box").hide();
           $("#frequently-searched-box").hide();
+          
         } else if (!$("body").hasClass("top-down") && keyCode == 13) {
           $(".default-data-show-not-loaded").hide();
-          _self.vars.enterIsClicked = true;
+          _self.vars.enterIsClicked = true; // convert text to selected language //
+          var searchText =  $('body').hasClass('top-down') ? $('.search-top-down').val() : $('.bottom-up-search').val();
+          if(searchText == 'language=en' || searchText == 'language=ja' || searchText == 'language=ko'){
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js:5784]: @BharadwajAd-Kore : comment this code which is only for testing, i  placed
@SunilSi-kore

File: src/components/search/findly-sdk.js [hunk 11]
```diff
@@ -7482,7 +7502,7 @@ FindlySDK.prototype.handleSearchRes = function (res) {
       // } 
       if (!$('body').hasClass('top-down')) {
         setTimeout(() => {
-          if ($('.messageBubble .userMessage span').last().text() == _self.vars.searchObject.searchText && $('.messageBubble .messageBubble-content .botMessage span:nth-child(2)').last().text() === 'Here are the relevant results') {
+          if ($('.messageBubble .userMessage span').last().text() == _self.vars.searchObject.searchText && $('.messageBubble .messageBubble-content .botMessage span:nth-child(2)').last().text() === langTranslator("sa_sdk_sure_please_find_the_matched_results_below")) {
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js]: @BharadwajAd-Kore : message was changed to 'Here are the relevant results' could you please check and change it revelant i18n key
@SunilSi-kore

File: src/components/search/findly-sdk.js [hunk 12]
```diff
@@ -17826,7 +17863,7 @@ FindlySDK.prototype.facetFilterTop = function () {
                       {{/if}}\
                   {{/each}}\
                   {{if searchFacet.multiselect}}\
-                  <div class="apply-btn">Apply</div>\
+                  <div class="apply-btn sdk-i18n-lang" sdk-i18n-key="sa_sdk_apply">{{html langTranslator("sa_sdk_apply")}}</div>\
                   {{/if}}\
                     {{if !searchFacet.multiselect}}\
                     <div class="clear-all sdk-clear-all-facet-top">Clear</div>\
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js:17869]: @BharadwajAd-Kore : i18n change missed
@SunilSi-kore

File: src/components/search/findly-sdk.js [hunk 13]
```diff
@@ -17858,8 +17895,8 @@ FindlySDK.prototype.fullResultRanking = function () {
                           <span class="hide {{if record.config.visible == true}}display-block{{else}}display-none{{/if}}">\
                               Hide\
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js:17896]: @BharadwajAd-Kore :hide  i18n change missed
@SunilSi-kore

File: src/components/search/findly-sdk.js [hunk 14]
```diff
@@ -17869,8 +17906,8 @@ FindlySDK.prototype.fullResultRanking = function () {
                           <span class="unpin {{if record.config.pinIndex >= 0}}display-block{{else}}display-none{{/if}}">\
                             UnPin\
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js:17907]: @BharadwajAd-Kore :unpin  i18n change missed
@SunilSi-kore

File: src/components/search/findly-sdk.js [hunk 15]
```diff
@@ -17894,22 +17931,21 @@ FindlySDK.prototype.fullResultRanking = function () {
...
+                    <span class="count boosted">${record.config.boost}X <span class="sdk-i18n-lang" sdk-i18n-key="sa_sdk_boosted_caps">{{html langTranslator("sa_sdk_boosted_caps")}}</span></span>\
+                  </div>\
+                  <div class="appearences-count customize-chips bg-data record-status-lowered {{if record.config.boost < 1}}display-block{{/if}}">\
+                    <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAMCAYAAABbayygAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAADJSURBVHgBdZDRDYIwEIZbEFKDTRiBEVzBSfDRmBRxA11BX3xTJ3AE4ghOgE7ggyaalvbskZCUWP633n253neUC31USm6/h/GdeJKWkDXG5AFQ84jiuGKLT+aDNJiKEEPbwqRQG17o2oUR4itdYw/ftGtggZIgV1LOGGMEJwGY83sX9UEXxqoLeYNwImT51+DCXHwiXZKlnOJlrLW+DVkjFIxCaw3XQet2UqGfXDTzQeswCNN2EsD6tR+demDP2p7RhfzLW+PuOzc/5PRxOXt0QzUAAAAASUVORK5CYII=">\
+                    <span class="count lowered">${record.config.boost}X <span class="sdk-i18n-lang" sdk-i18n-key="sa_sdk_lowered_caps">{{html langTranslator("sa_sdk_lowered_caps")}}</span></span>\
+                  </div>\
                     <div class="tag-ref">FAQ Response</div>\
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js:17949]: @BharadwajAd-Kore : FAQ Response i18n change missed
@SunilSi-kore

File: src/components/search/findly-sdk.js [hunk 16]
```diff
@@ -20717,7 +20753,7 @@ FindlySDK.prototype.searchHistroy = function (findlyConfig) {
             messageData.isFromHistory = true;
             //messageData.timestamp = history.timestamp;
           } else {
-            messageData.text = 'Here are the relevant results';
+            messageData.text = langTranslator("sa_sdk_sure_please_find_the_matched_results_below");
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js:20756]: @BharadwajAd-Kore : message was changed to 'Here are the relevant results' could you please check and change it revelant i18n key
@SunilSi-kore

File: src/components/search/findly-sdk.js [hunk 17]
```diff
@@ -20717,7 +20753,7 @@ FindlySDK.prototype.searchHistroy = function (findlyConfig) {
...
             messageData.from = 'bot';
             messageData.count = _self.countTotalResults(history.response.message[0].component.payload.template, 0);
             messageData.text = messageData.count?'Sure, please find the matched results below':'No results were found for this query';
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js:20759]: @BharadwajAd-Kore : message was changed to 'Here are the relevant results', could you please check and change it revelant i18n key and also 'No results were found for this query' i18n change missed
@SunilSi-kore

File: src/components/search/searchTemplate.js [hunk 18]
```diff
@@ -15,23 +15,23 @@
                 <div class="total-structured-data-wrap {{if viewType=="Customize"&&devMode==true}}{{if isFullResults == true}}customization{{/if}}{{/if}} {{if maxSearchResultsAllowed ==0}}display-none{{/if}}">\
                   {{if tour && isFullResults == true && viewType=="Customize"&&devMode==true}}\
```
Comment (VenkatIc-Kore) [inline src/components/search/searchTemplate.js:16]: @BharadwajAd-Kore : Not required to change i18n changes , we are not using this file
@SunilSi-kore

File: src/templatemanager/templates/feedBackFormTemplate/feedBackFormTemplate.ts [hunk 19]
```diff
@@ -24,27 +25,27 @@ class FeedBackFormTemplate {
...
+          <button class="temp-feed-back-ans-tag-btn" value="outdated"><span class="sdk-i18n-lang"  sdk-i18n-key="sa_sdk_outdated">{{html langTranslator("sa_sdk_outdated")}}</span></button>\
+          <button class="temp-feed-back-ans-tag-btn" value="few_results"><span class="sdk-i18n-lang"  sdk-i18n-key="sa_sdk_few_results">{{html langTranslator("sa_sdk_few_results")}}</span></button>\
+          <button class="temp-feed-back-ans-tag-btn" value="other"><span class="sdk-i18n-lang"  sdk-i18n-key="sa_sdk_other">{{html langTranslator("sa_sdk_other")}}</span></button>\
       </div>\
-      <div class="temp-feed-back-opt-qns">Please help us with more details (Optional)</div>\
+      <div class="temp-feed-back-opt-qns"><span class="sdk-i18n-lang"  sdk-i18n-key="sa_sdk_please_help_with_more_details">{{html langTranslator("sa_sdk_please_help_with_more_details")}}</span></div>\
       <div class="temp-feed-back-opt-ans">\
           <textarea id="feedback-input-text" placeholder="Start typing here..."></textarea>\
```
Comment (VenkatIc-Kore) [inline src/templatemanager/templates/feedBackFormTemplate/feedBackFormTemplate.ts]: @BharadwajAd-Kore : placeholder i18n change missed
@SunilSi-kore

**Review by VenkatIc-Kore — COMMENTED** [review]
please fix those commented lines

File: src/components/search/findly-sdk.js (hunk 9 above)
Comment (BharadwajAd-Kore) [reply src/components/search/findly-sdk.js:1559]: updated.

File: src/components/search/findly-sdk.js (hunk 8 above)
Comment (BharadwajAd-Kore) [reply src/components/search/findly-sdk.js:20]: updated.

File: src/components/search/findly-sdk.js (hunk 10 above)
Comment (BharadwajAd-Kore) [reply src/components/search/findly-sdk.js:5784]: updated.

File: src/components/search/findly-sdk.js (hunk 16 above)
Comment (VenkatIc-Kore) [reply src/components/search/findly-sdk.js:20756]: @BharadwajAd-Kore : Please change 'Sure, please find the matched results below' into 'Here are the relevant results'
@SunilSi-kore

File: src/components/search/findly-sdk.js [hunk 20]
```diff
@@ -20717,10 +20750,10 @@ FindlySDK.prototype.searchHistroy = function (findlyConfig) {
             messageData.isFromHistory = true;
             //messageData.timestamp = history.timestamp;
           } else {
-            messageData.text = 'Here are the relevant results';
+            messageData.text = langTranslator("sa_sdk_sure_please_find_the_matched_results_below");
```
Comment (VenkatIc-Kore) [inline src/components/search/findly-sdk.js:20753]: @BharadwajAd-Kore : Please replace this sa_sdk_sure_please_find_the_matched_results_below with 'Here are the relevant results' in i18n sdk key
@SunilSi-kore

File: src/templatemanager/templates/finalResultsTemplate/finalResultsTemplate.ts (hunk 2 above)
Comment (VenkatIc-Kore) [reply src/templatemanager/templates/finalResultsTemplate/finalResultsTemplate.ts]: @BharadwajAd-Kore : still ANSWERED BY AI i18n changes are missed
@SunilSi-kore

File: src/templatemanager/templates/finalResultsTemplate/finalResultsTemplate.ts (hunk 3 above)
Comment (VenkatIc-Kore) [reply src/templatemanager/templates/finalResultsTemplate/finalResultsTemplate.ts]: @BharadwajAd-Kore : still ANSWERED BY AI i18n changes are  missed
@SunilSi-kore

File: src/templatemanager/templates/fullsearchResultsTemplate/fullsearchResultsTemplate.ts (hunk 4 above)
Comment (VenkatIc-Kore) [reply src/templatemanager/templates/fullsearchResultsTemplate/fullsearchResultsTemplate.ts]: @BharadwajAd-Kore : still Clear i18n changes are  missed
@SunilSi-kore

File: src/templatemanager/templates/searchListViewTemplate/searchListViewTemplate.ts (hunk 5 above)
Comment (VenkatIc-Kore) [reply src/templatemanager/templates/searchListViewTemplate/searchListViewTemplate.ts]: @BharadwajAd-Kore : still customize i18n change missed
@SunilSi-kore

File: src/templatemanager/templates/snippetListTemplate/snippetListTemplate.ts (hunk 6 above)
Comment (VenkatIc-Kore) [reply src/templatemanager/templates/snippetListTemplate/snippetListTemplate.ts]: @BharadwajAd-Kore : still ANSWERED BY AI key don't have i18n changes , could u please check and replace i18n change in all the places
@SunilSi-kore

File: src/templatemanager/templates/snippetParagraphTemplate/snippetParagraphTemplate.ts (hunk 7 above)
Comment (VenkatIc-Kore) [reply src/templatemanager/templates/snippetParagraphTemplate/snippetParagraphTemplate.ts]: @BharadwajAd-Kore : still ANSWERED BY AI i18n changes are  missed
@SunilSi-kore

**Review by VenkatIc-Kore — COMMENTED** [review]
Could you please fix the commented lines

**Review by VenkatIc-Kore — APPROVED** [review]
looks good

**Review by SunilSi-kore — APPROVED** [review]

**Comment by BharadwajAd-Kore** [comment]
Merging the i18 changes

**Pull request merged**